Hackney.Shared.HousingSearch/Gateways/Models/Tenures/QueryableTempAccommodationInfo.cs

```diff
@@ -1,7 +1,10 @@
+using Hackney.Shared.HousingSearch.Domain.Tenure;
+
 namespace Hackney.Shared.HousingSearch.Gateways.Models.Tenures
 {
     public class QueryableTempAccommodationInfo
     {
         public string BookingStatus { get; set; }
+        public TempAccommodationOfficer AssignedOfficer { get; set; }
```

Comment:
I think this should be `QueryableTempAccommodationOfficer` rather than `TempAccommodationOfficer`.

The queryable model should contain only other queryable models.

Reply:
Addressed in [cebad9e](https://github.com/LBHackney-IT/housing-search-shared/pull/74/commits/cebad9ecced7120c18e3bbcd9cd2d26a571dc601)